libpod/rootless_cni_linux.go

```diff
@@ -22,10 +22,10 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
+// Built from ../contrib/rootless-cni-infra.
+// Latest version: 1
 var rootlessCNIInfraImage = map[string]string{
-	// Built from ../contrib/rootless-cni-infra
-	// TODO: move to Podman's official quay
-	"amd64": "ghcr.io/akihirosuda/podman-rootless-cni-infra:gd34868a13-amd64",
+	"amd64": "quay.io/libpod/rootless-cni-infra@sha256:2371c5ad4dee37e4519ac5dcf234fc96001be0e9dad901a4b9cfe0a320e88e2e",
```

Comment:
tag?

Reply:
Didn't you ask for digest in https://github.com/containers/podman/pull/7625#discussion_r487744211?

Reply:
Ah, I meant specifying both tag and digest. We should still have the tag for clarity to humans.

Reply:
Unfortunately, Podman does not support using `:tag@digest`. Shall we add a comment behind?

Reply:
:eyes:

> Shall we add a comment behind?

Yes please